src/runtime/components/Tabs.vue

```diff
@@ -73,7 +76,7 @@ const ui = computed(() => tv({ extend: tabs, slots: props.ui })({
 
       <TabsTrigger v-for="(item, index) of items" :key="index" :value="item.value || String(index)" :disabled="item.disabled" :class="ui.trigger()">
         <slot name="leading" :item="item" :index="index">
-          <UAvatar v-if="item.avatar" size="2xs" v-bind="item.avatar" :class="ui.leadingAvatar()" />
+          <UAvatar v-if="item.avatar" :size="ui.avatarSize()" v-bind="item.avatar" :class="ui.leadingAvatar()" />
```

Comment:
Good idea to put this in the config but I guess this should be named `ui.leadingAvatarSize()`.